report/thesis.tex

```diff
@@ -154,38 +177,36 @@ \chapter{Tendermint protocol overview and the fork accountability problem}
 
 ### Definition
 
-A fork is a split in the blockchain network or, in other words, a divergence into different block paths. 
-A **fork** happens when two correct validators decide on different blocks in the same height (i.e., there are two commits for different blocks in the same consensus instance). 
+A **fork** is a split in the blockchain network or, in other words, a divergence into different block paths. 
+A fork happens when two correct validators decide on different blocks in the same height (i.e., there are two commits for different blocks in the same consensus instance). 
 
-In the context of the Tendermint protocol, a fork happens in height *h* when a quorum of messages (*2f + 1* PRECOMMIT messages) are sent for different values: more formally, there are two sets of *2f + 1* PRECOMMIT messages, A and B, such that A's messages are PRECOMMIT votes for value *v* and B's messages are PRECOMMIT votes for value *v' != v* in height *h*.  
+In the context of the Tendermint protocol, a fork happens in height *h* when a quorum of messages (at least *2f + 1* PRECOMMIT messages) are sent for different values: more formally, there are two sets of at least *2f + 1* PRECOMMIT messages, A and B, such that A's messages are PRECOMMIT votes for value *v* and B's messages are PRECOMMIT votes for value *v' != v* in height *h*.  
 
-The Tendermint conensus algorithm guarantees that a fork (i.e., a violation of the agreement property) cannot happen when the number of faulty processes is less than one third.
-We want to address and analyze the problem of defining a Tendermint consensus algorithm that provides the same four properties (validity, agreement, termination, integrity) when the number of faulty processes is less than one third and guarantees, in addition, an efficient detection of faulty processes that can potentially make the consensus fail with a fork when there are more than one third of faulty validators.
+The Tendermint consensus algorithm guarantees that a fork (i.e., a violation of the agreement property) cannot happen when the number of faulty processes is less than one third.
+We want to address and analyze the problem of defining a Tendermint consensus algorithm that provides the same four properties (validity, agreement, termination, integrity) when the number of faulty processes is less than one third and guarantees, in addition, an efficient detection of faulty processes responsible of a fork when there are more than one third of faulty validators.
 
-In particular, we assume there are more than one third of faulty validators such that the following holds:
+In particular, we assume there are more than one third of faulty validators such that the following holds:[^1]
 
     f < num. faulty processes <= 2f
 
+[^1]: Fork accountability cannot be solved when the number of faulty proccesses is greater than *2f*: in this case, faulty processes basically control the system and nothing can be done to solve the problem.
```

Comment:
Not true. If every message has its justification, then it is.

Reply:
We should remove this

Reply:
Yeah we already said that, it is now here again because I moved some parts in other chapter of the report and, in fact, this was previously part of the discussion about the piggybacking solution which now I mention in the Future work chapter. I'll remove this for simplicity.